src/java/main/org/apache/zookeeper/server/persistence/FileTxnLog.java

```diff
@@ -199,14 +199,16 @@ public synchronized boolean append(TxnHeader hdr, Record txn)
                 LOG.warn("Current zxid " + hdr.getZxid()
```

Comment:
I prefer return early too, I didn't change it as it's not related with this diff, and I'd like to make it with minimum change. I can update it here too if you guys prefer.